portal/apps/org.wso2.carbon.iam.user-portal.app/pom.xml

```diff
@@ -76,7 +76,9 @@
                         </goals>
                         <configuration>
                             <instructions>
-                                <Import-Package>org.wso2.is.portal.user.client.api.*</Import-Package>
+                                <Import-Package>
+                                    org.wso2.is.portal.user.client.api.util, org.wso2.is.portal.user.client.api.*
```

Comment:
no need to import util explicitly since we import org.wso2.is.portal.user.client.api.*